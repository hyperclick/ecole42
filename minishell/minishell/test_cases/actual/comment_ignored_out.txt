/mnt/c/git/e47/minishell/minishell
