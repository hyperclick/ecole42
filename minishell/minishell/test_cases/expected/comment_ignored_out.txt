/Users/darugula/git/e57/minishell/minishell
